third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -631,7 +631,10 @@ function isBlinkObject(x) {
 function isBlinkInstanceOf(x, target) {
   return isBlinkObject(x) &&
     target?.name &&
-    hasInProtoChain(x.constructor, target.name);
+    hasInProtoChain(
```

Comment:
This line was causing `Blocked a frame with origin` exceptions. That is low priority, but separating them helps us determine who is causing it.